component-runtime-testing/component-runtime-http-junit/src/test/java/org/talend/sdk/component/junit/http/junit4/CaptureJUnit4HttpApiTest.java

```diff
@@ -118,7 +117,7 @@ public void evaluate() throws Throwable {
             final String lines = String.join("\n", Files.readAllLines(output));
             assertJSONEquals("[\n" + "  {\n" + "    \"request\":{\n" + "      \"headers\":{\n"
                     + "        \"content-length\":\"0\",\n"
-                    + "        \"Accept\":\"*/*\",\n"
```

Comment:
Should it be removed ? Was it related to java17 ?

Reply:
changes in gttp client AFAIR